server/config/production.yml

```diff
@@ -19,7 +19,9 @@ defaultSettings:
   canTutorExcuseStudents: true
   defaultTeamSize: 3
 
-# Settings used for the puppeteer instance
-puppeteer:
+# Settings used for the gotenberg instance
+gotenberg:
+  host: 'http://localhost'
```

Comment:
you sure?
in production this should be deployed via docker / k8s, right?
shouldn't this then be sth like gotenberg or so (depending on one names the docker container)?
see above for database, there it is named tms_sql

Reply:
I adapted the files, hopefully as expected. 
There are also mentions of puppeteer in the dockerfile and some docs files that reference build-test-docker/buildAndTestDocker.ts script as it existed only because of puppeteer. Should it be removed?

Reply:
tbh lets just get rid of it